### DISCUSSION TITLE
Failed partials-missing signage points

### DISCUSSION THREAD

**Discussion by Stegini777** [Chia-Network/chia-blockchain#19070] [Support]
Please if someone can help me with this...
![Στιγμιότυπο οθόνης από 2024-12-19 00-04-47](https://github.com/user-attachments/assets/9a91a242-3501-4022-913c-9049c9846776)

**Comment by BrandtH22** [comment]
Hey @Stegini777 , as a reminder we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

As for the errors it looks like you might have some networking issues. Missing signage points means you did not receive the signage points from peers so either you are not connecting to enough peers or there is an issue with those connections. 

As for invalid partials, since you are not getting the correct or any signage points from peers the partials for the pool are invalid (due to the same networking issues more than likely).

- Can you provide your log file from `~/.chia/mainnet/log/debug.log`?
- Also can you click on the node sync icon in the top right and see how many peers you are connected to?

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
